content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,397 @@
+
+You can also request pod eviction manually using the `kubectl drain` command
+or the [Eviction API](/docs/reference/generated/kubernetes-api/v1.21/#create-eviction-pod-v1-core). 
+
+## Eviction signals {#eviction-signals}
+
+Eviction signals are the current state of a particular resource at a specific
+point in time. Kubelet uses eviction signals to make eviction decisions by
+comparing the signals to eviction thresholds, which are the minimum amount of 
+the resource that should be available on the node. 
+
+Kubelet uses the following eviction signals:
+
+| Eviction Signal      | Description                                                                           |
+|----------------------|---------------------------------------------------------------------------------------|
+| `memory.available`   | `memory.available` := `node.status.capacity[memory]` - `node.stats.memory.workingSet` |
+| `nodefs.available`   | `nodefs.available` := `node.stats.fs.available`                                       |
+| `nodefs.inodesFree`  | `nodefs.inodesFree` := `node.stats.fs.inodesFree`                                     |
+| `imagefs.available`  | `imagefs.available` := `node.stats.runtime.imagefs.available`                         |
+| `imagefs.inodesFree` | `imagefs.inodesFree` := `node.stats.runtime.imagefs.inodesFree`                       |
+| `pid.available`      | `pid.available` := `node.stats.rlimit.maxpid` - `node.stats.rlimit.curproc`           |
+
+In this table, the `Description` column shows how kubelet gets the value of the
+signal. Each signal supports either a percentage or a literal value. Kubelet 
+calculates the percentage value relative to the total capacity associated with
+the signal. 
+
+The value for `memory.available` is derived from the cgroupfs instead of tools
+like `free -m`. This is important because `free -m` does not work in a
+container, and if users use the [node
+allocatable](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable) feature, out of resource decisions
+are made local to the end user Pod part of the cgroup hierarchy as well as the
+root node. This [script](/examples/admin/resource/memory-available.sh)
+reproduces the same set of steps that the `kubelet` performs to calculate
+`memory.available`. The `kubelet` excludes inactive_file (i.e. # of bytes of
+file-backed memory on inactive LRU list) from its calculation as it assumes that
+memory is reclaimable under pressure.
+
+The kubelet supports the following filesystems:
+
+1. `nodefs`: Volumes, daemon logs, etc.
+1. `imagefs`: Container runtimes use `imagefs` to store container images and
+   container writable layers. This is optional.
+
+Kubelet auto-discovers these filesystems using cAdvisor and ignores other
+filesystems. Kubelet does not support other configurations. For example, do not
+store volumes and logs in a dedicated filesystem.
+
+{{<note>}}
```

Comment:
Is this still a "future release" feature or has garbage collection been deprecated?